package.json

```diff
@@ -18,8 +18,9 @@
     }
   ],
   "dependencies": {
-    "msgpack5rpc": "^1.0.0",
-    "cson": "^2.0.0"
+    "cson": "^2.0.0",
+    "lodash": "^3.10.0",
```

Comment:
Seems that only one functions from `lodash` is used, adding introducing a new dependency here would be an overkill.